DirectProgramming/C++SYCL_FPGA/Tutorials/Features/IPAuthoring_interfaces_overview/handshake-control/Readme.md

```diff
@@ -0,0 +1,206 @@
+# Handshake-control
+This example demonstrate a streaming pipelined kernel with conduit data interfaces and implementing a simple vector addition.
+
+This design will show how to specify streaming invocation interface through kernel properties and how to specify conduit interface using `annotated_ptr`.
+
+## Invocation Interface - Streaming pipelined interface
+In this design, the `SimpleVAdd` kernel is using streaming pipelined invocation interface. This is explicitly specified though kernel properties. 
+
+To use kernel properties, include either of the following header files:
+```cpp
+#include <sycl/ext/intel/fpga_extensions.hpp>
+```
+Or
+```cpp
+#include <sycl/ext/intel/experimental/fpga_kernel_properties.hpp>
+```
+In this design following member functions is added to the kernel functor to specify kernel properties:
+```cpp
+auto get(sycl::ext::oneapi::experimental::properties_tag) {
+    return sycl::ext::oneapi::experimental::properties{
+        sycl::ext::intel::experimental::
+            streaming_interface_accept_downstream_stall,
+        sycl::ext::intel::experimental::pipelined<1>};
+  }
+```
```

Comment:
```suggestion
In this design, the following member function is added to the kernel functor to specify kernel properties:
auto get(sycl::ext::oneapi::experimental::properties_tag) {
    return sycl::ext::oneapi::experimental::properties{
        sycl::ext::intel::experimental::
            streaming_interface<>,
        sycl::ext::intel::experimental::pipelined<1>};
  }
```

as of the latest 2024.0, you can use `streaming_interface<>` instead of `streaming_interface_...blahblahblah` which is more concise and easier to read. `streaming_interface<>` defaults to `streaming_interface_downstream_stall`